0.1.0/README.md

```diff
@@ -0,0 +1,26 @@
+# UGent Master Thesis Template
+
+This Typst template is made for a master thesis at UGent.
+The authors of this template are not affiliated with UGent.
+
+## Usage
+
+Run the following command in your terminal. The font path needs to be supplied as an argument if 
+the UGent fonts are not installed system-wide. This command will create the output at `main.pdf`.
+
+```shell
+typst watch main.typ --font-path fonts
+```
+
+## Dependencies
+
+- The library [acrostiche](https://typst.app/universe/package/acrostiche) is used to manage acronyms.
+    These are maintained under `src/acronyms.typ`.
+
+## Other remarks
+
+The `front-page` function is really only a placeholder for the eventual front page which you will have to export
+from Plato yourself.
```

Comment:
Maybe just leave the front page out, might lead to someone not noticing it isn't the correct front page. Would be a painful mistake to make.